src/user-federation/KerberosSettingsCache.tsx

```diff
@@ -1,16 +1,60 @@
-import { Form, FormGroup, Select, SelectOption } from "@patternfly/react-core";
+import {
+  Form,
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectVariant,
+  TextInput,
+} from "@patternfly/react-core";
 import { useTranslation } from "react-i18next";
-import React from "react";
 import { HelpItem } from "../components/help-enabler/HelpItem";
+import React, { useState } from "react";
+import { useForm, Controller } from "react-hook-form";
+import { UserFederationKerberosCacheRepresentation } from "./models/user-federation";
 
 export const KerberosSettingsCache = () => {
   const { t } = useTranslation("user-federation");
   const helpText = useTranslation("user-federation-help").t;
 
+  const [isCachePolicyDropdownOpen, setIsCachePolicyDropdownOpen] = useState(
+    false
+  );
+  const [isEvictionHourDropdownOpen, setIsEvictionHourDropdownOpen] = useState(
+    false
+  );
+  const [
+    isEvictionMinuteDropdownOpen,
+    setIsEvictionMinuteDropdownOpen,
+  ] = useState(false);
+  const [isEvictionDayDropdownOpen, setIsEvictionDayDropdownOpen] = useState(
+    false
+  );
+
+  const { handleSubmit, control, register } = useForm<
+    UserFederationKerberosCacheRepresentation
+  >();
+  const onSubmit = (data: UserFederationKerberosCacheRepresentation) => {
+    console.log(data);
+  };
+
+  const hourOptions = [
+    <SelectOption key={0} value="Choose..." isPlaceholder />,
+  ];
+  for (let index = 0; index < 24; index++) {
+    hourOptions.push(<SelectOption key={index + 1} value={index} />);
+  }
+
+  const minuteOptions = [
+    <SelectOption key={0} value="Choose..." isPlaceholder />,
+  ];
+  for (let index = 0; index < 60; index++) {
+    minuteOptions.push(<SelectOption key={index + 1} value={index} />);
```

Comment:
From a user’s perspective, I think it would make more sense to start at 1 (rather than 0) here. Can 

```suggestion
    minuteOptions.push(<SelectOption key={index + 1} value={index + 1} />);
```